bizdemo/easy_note/cmd/note/Makefile

```diff
@@ -1,2 +1,2 @@
 server:
-	kitex -service demonote ../../idl/note.thrift
+	kitex -service demonote -use github.com/cloudwego/kitex-examples/biz-demo/easy_note/kitex_gen ../../idl/note.thrift
```

Comment:
it will cause `kitex_gen is not generated due to the -use option`, generate `kitex_gen` without `-service` first, then use the `-use`

Reply:
The command in this makefile is used to generate rpc server，and the command without -use option to generate rpc client is in the bizdemo/cmd/api/Makefile

Reply:
> it will cause `kitex_gen is not generated due to the -use option`, generate `kitex_gen` without `-service` first, then use the `-use`

The command in this makefile is used to generate rpc server，and the command without -use option to generate rpc client is in the bizdemo/cmd/api/Makefile

Reply:
> > it will cause `kitex_gen is not generated due to the -use option`, generate `kitex_gen` without `-service` first, then use the `-use`
> 
> The command in this makefile is used to generate rpc server，and the command without -use option to generate rpc client is in the bizdemo/cmd/api/Makefile

My fault, sorry.